### ISSUE TITLE
possible to ignore buffers when doing helm-ag--buffer-search?

### ISSUE THREAD

**Issue by mmcnl** [syohex/emacs-helm-ag#269]
- Emacs version
25.1
- Operating system
OSX
- Search command(ag, pt, ack etc) and version
ag version 0.32.0
- Last value of `helm-ag--last-command`
```
"ag" "--hidden" "--nocolor" "--nogroup" "--smart-case" "defun" "/Users/matt/.emacs.d/elpa/helm-ag-20160917.2126/helm-ag.el" "/Users/matt/.emacs.d/elpa/helm-20160919.2219/helm-buffers.el" "/Users/matt/.emacs.d/init.el" "/Users/matt/.emacs.d/TAGS")
```

### Actual behavior
When using `helm-ag--buffer-search`, Helm-ag returns results from boring files (like `TAGS`)

### Expected behavior
I'd love to be able to skip buffers (like I can skip files) with a regex. Since helm-ag just uses an unfiltered `buffer-list` to generate the files for `ag` to search
https://github.com/syohex/emacs-helm-ag/blob/master/helm-ag.el#L197

, it seems like there is no way to override this (for example by passing extra options to `ag`).

**Comment by syohex** [comment]
It is impossible now. I'll add such custom variable, please wait.

**Comment by syohex** [comment]
I have merged #270. Please check latest version.

**Comment by mmcnl** [comment]
Thanks @syohex for the quick reply!
